nautobot/ipam/tables.py

```diff
@@ -79,7 +79,7 @@
 {% elif perms.ipam.add_ipaddress %}
     <a href="\
 {% url 'ipam:ipaddress_add' %}\
-?address={{ record.1 }}\
+?address={{ record.1 }}&namespace={{ object.namespace.pk }}\
 {% if object.vrf %}&vrf={{ object.vrf.pk }}{% endif %}\
 {% if object.tenant %}&tenant={{ object.tenant.pk }}{% endif %}\
 " class="btn btn-xs btn-success">\
```

Comment:
It is a little confusing that we are using both `record` and `object` here. I wonder if we can unify them.
Nonetheless, I feel like checking for `object.vrf` is not correct here as it's a ForeignKey field that only exists on the `VRFPrefixAssignment` model and not on the `Prefix` model itself.

Reply:
There's a comment earlier in the file, `record` is the row of the table (the specific `IPAddress`) and `object` is the object we're seeing the detail view of (the parent `Prefix`).

I think you're right that `object.vrf` will never evaluate in either of these cases after the 2.0 IPAM changes - want to open a separate issue about that?